README.tpl

```diff
@@ -9,6 +9,7 @@
 [std::io::Read]: https://doc.rust-lang.org/stable/std/io/trait.Read.html
 [Agent]: https://docs.rs/ureq/latest/ureq/struct.Agent.html
 [get()]: https://docs.rs/ureq/latest/ureq/fn.get.html
+[patch()]: https://docs.rs/ureq/latest/ureq/fn.patch.html
```

Comment:
This line is not needed. The other entries in this list are things that are linkified in the lib.rs doccomment and are normally provided by rustdoc's intra-doc comments, but need to be copied here for when they are rendered by GitHub Markdown.

Reply:
Ok, will remove and force-push.

Reply:
Done.